Sources/MockoloFramework/Models/Import.swift

```diff
@@ -15,10 +15,10 @@
 //
 
 /// A structure defining an "import" statement parsed by `Generator`, including various modifiers.
-struct Import: CustomStringConvertible {
+public struct Import: CustomStringConvertible {
```

Comment:
[self review]

Because other types such as `ImportMap` are public scope, `Import` should be public as well.

Reply:
`ImportMap` (and `SourceParser`)could be `internal`. 

Rather than promoting `public`, we should focus on correcting existing flawed `public` implementations.

Reply:
Thank you!

Because this is not crucial, I will address it after your other reviews are finished.

Reply:
This PR contains unnecessary changes because of that, so it would be appreciated if you could fix them.